pom.xml

```diff
@@ -50,6 +52,31 @@
             <version>1.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
do we need joda for this plugin?

Reply:
I don't know how to solve the plugin conflict in another way:

Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for joda-time:joda-time:2.1 paths to dependency are:
+-org.jenkins-ci.plugins:sidebar-link:2.00.1
  +-org.jenkinsci.plugins:pipeline-model-definition:1.8.4
    +-org.jenkinsci.plugins:pipeline-model-api:1.8.4
      +-com.github.fge:json-schema-validator:2.0.4
        +-joda-time:joda-time:2.1
and
+-org.jenkins-ci.plugins:sidebar-link:2.00.1
  +-io.jenkins.configuration-as-code:test-harness:1.47
    +-com.github.erosb:everit-json-schema:1.12.1
      +-com.damnhandy:handy-uri-templates:2.1.8
        +-joda-time:joda-time:2.10.2